wpimath/src/test/java/edu/wpi/first/math/geometry/Pose3dTest.java

```diff
@@ -14,7 +15,7 @@
 import org.junit.jupiter.api.Test;
 
 class Pose3dTest {
-  private static final double kEpsilon = 1E-9;
+  private static final double kEpsilon = 1E-8;
```

Comment:
Do the tests fail without the wider tolerance?

Reply:
`./gradlew wpimath:build` runs all the C++ and Java tests. You can add `--continue` so it runs all the tasks it can before exiting, even if some of them fail.

Reply:
But if Java fails with 1e-9, I'm fine with having both sets of tests use 1e-8.